cmds/info.c

```diff
@@ -229,6 +229,10 @@ static int fill_cmdline(void *arg)
 	for (i = 0, p = buf; i < ret; i++, p++) {
 		if (*p == '\0')
 			*p = ' ';
+		if (*p == '\n') {
+			memcpy(p, p + 1, ret - i);
+			p--;
+		}
```

Comment:
Why not simply changing it as a normal whitespace?

Reply:
It is my mistake.
```
$ uftrace ./test "a\
b"
```
I wrongly thought it should be handled the same way as above.
So I tried to remove the '\n' and join the string.
```
$ uftrace ./test "a
b"
```
However, in this case, the strings are separated.
I fixed it to be handled the same way as white space.